src/wp-includes/js/admin-bar.js

```diff
@@ -129,6 +194,15 @@ if ( typeof(jQuery) != 'undefined' ) {
 			target.siblings('.ab-sub-wrapper').find('.ab-item').each(refresh);
 		});
 
+		/**
+		 * @summary Scrolls to top of page by clicking the adminbar.
```

Comment:
See `@access private` comment from above. Applies to all anonymous functions in this file. I'm not sure if they should receive the private tag.. please check